.ci_support/win_64_numpy1.26python3.12.____cpython.yaml

```diff
@@ -12,6 +14,8 @@ pin_run_as_build:
     max_pin: x.x
 python:
 - 3.12.* *_cpython
+pytorch:
+- '2.4'
```

Comment:
As far as I can tell, pytorch v 2.4 (or any modern version, really) isn't built by conda-forge for windows. Did you manage to get this working? The CI builds didn't pass for Windows...